### PR TITLE
[ASM] xpsupport-2404

### PR DESCRIPTION
**Pull request by johnnywilkes**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
https://jira-dc.paloaltonetworks.com/browse/XPSUPPORT-2404

## Description
Added a check to determines if the *Cortex Attack Surface Management* integration instance is configured.

## Must have
- [ ] Tests
- [X] Documentation

### CODE REVIEW

**Comment by johnnywilkes** [comment]
@ShirleyDenkberg , please review

@BigEasyJ / @capanw , please review

File: Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_7_25.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+#### Playbooks
+
+##### Cortex ASM - Remediation Guidance
+
+- Added a check to determines if the *Cortex Attack Surface Management* integration instance is configured.
```
Comment (BigEasyJ) [inline Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_7_25.md:6]:
```suggestion
Added a check to determines if the *Cortex Attack Surface Management* integration instance is configured.
```

**Review by BigEasyJ — APPROVED** [review]

**Comment by johnnywilkes** [comment]
@YuvHayun / @melamedbn , this has been approved by Xpanse.  Please merge when possible

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Remediation_Guidance_README.md [hunk 2]
```diff
@@ -10,15 +10,16 @@ This playbook does not use any sub-playbooks.
 
 ### Integrations
 
-This playbook does not use any integrations.
+* Cortex Attack Surface Management
```
Comment (ShirleyDenkberg) [inline Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Remediation_Guidance_README.md:13]:
```suggestion
Cortex Attack Surface Management
```

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Remediation_Guidance_README.md [hunk 3]
```diff
@@ -10,15 +10,16 @@ This playbook does not use any sub-playbooks.
...
 
 ### Scripts
 
-Set
+* Set
```
Comment (ShirleyDenkberg) [inline Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Remediation_Guidance_README.md:17]:
```suggestion
Set
```
When there is only one item, we do not bullet it.

File: Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_7_24.md [hunk 4]
```diff
@@ -0,0 +1,6 @@
+
+#### Playbooks
+
+##### Cortex ASM - Remediation Guidance
+
+Added a check to determine if the *Cortex Attack Surface Management* integration instance is configured.
```
Comment (ShirleyDenkberg) [inline Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_7_24.md:6]:
```suggestion
Added a check to determine if the **Cortex Attack Surface Management** integration instance is configured.
```

**Comment by ShirleyDenkberg** [comment]
@BigEasyJ @melamedbn @YuvHayun Doc review completed.

File: Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_7_24.md (hunk 4 above)
Comment (BigEasyJ) [reply Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_7_24.md]: Addressed in a1e7cfb

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Remediation_Guidance_README.md (hunk 3 above)
Comment (BigEasyJ) [reply Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Remediation_Guidance_README.md]: Addressed in a1e7cfb

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Remediation_Guidance_README.md (hunk 2 above)
Comment (BigEasyJ) [reply Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Remediation_Guidance_README.md]: Addressed in a1e7cfb

**Comment by BigEasyJ** [comment]
@YuvHayun @melamedbn please merge #32312 before if needed but the versions should not conflict now.

**Comment by BigEasyJ** [comment]
Ready to merge

**Comment by YuvHayun** [comment]
Merging to internal pr, thx for your contribution.

**Pull request merged**